code/stats/scoring.cpp

```diff
@@ -105,12 +105,10 @@ void parse_rank_tbl()
 		required_string("#End");
 
 		// be sure that all rank points are in order
-#ifndef NDEBUG
```

Comment:
Since the Warning below will only be seen in debug builds, removing this preprocessor directive isn't needed.